packages/paste-core/components/alert/package.json

```diff
@@ -30,6 +30,7 @@
     "@twilio-paste/design-tokens": "^8.0.0",
     "@twilio-paste/icons": "^9.0.0",
     "@twilio-paste/media-object": "^7.0.0",
+    "@twilio-paste/screen-reader-only": "^10.0.0",
```

Comment:
Note:  I noticed that ScreenReaderOnly isn't a dependency of Alert even tho its used in it, so I added it.

Reply:
Should this have a changeset?

Reply:
yeah technically. A missing peer dependency is kind of a big deal

Reply:
Cool! Added a patch changeset for alert